src/analyses/flagModes.ml

```diff
@@ -32,12 +32,12 @@ struct
               ctx.local
             end else begin
               match eval_int (Analyses.ask_of_ctx ctx) rval with
-              | Some ex -> D.add f (`Lifted (false,true,ex)) ctx.local
+              | Some ex -> D.add f (`Lifted (false,true, Option.get(IntDomain.FlatPureIntegers.to_int (IntOps.BigIntOps.to_int64 ex)))) ctx.local
```

Comment:
This seems complicated

Reply:
There's a very similar use below, where this is done more directly.